ui/redux/selectors/content.js

```diff
@@ -132,7 +132,7 @@ export const makeSelectIsText = (uri: string) =>
   createSelector(
     makeSelectMediaTypeForUri(uri),
     mediaType => {
-      const isText = ['text', 'document'].includes(mediaType);
+      const isText = ['document', 'script'].includes(mediaType);
```

Comment:
There are some files that will still have the `text` `mediaType`. Just add `script` to the end without removing `text`

Reply:
https://github.com/lbryio/lbry-desktop/pull/3511/commits/c063ee9cfb7428314a50dd53a6b886946068aaaf